applications/nrf5340_audio/src/audio/sw_codec_select.c

```diff
@@ -20,20 +21,70 @@ LOG_MODULE_REGISTER(sw_codec_select, CONFIG_SW_CODEC_SELECT_LOG_LEVEL);
 
 static struct sw_codec_config m_config;
 
+static struct sample_rate_converter_ctx encoder_converters[2];
```

Comment:
We need to define this and not use hard-coded values.

Reply:
AFAIK this is not changed.

Reply:
My bad, adding to next commit